C_routing.md

```diff
@@ -906,10 +896,10 @@ end
 
 Routing is a big topic, and we have covered a lot of ground here. The important points to take away from this guide are:
 - Routes which begin with an HTTP verb name expand to a single clause of the match function.
-- Routes which begin with 'resources' expand to 8 clauses of the match function.
+- Routes which begin with 'resources' expand to 8 clauses of the match function: GET, POST, PUT, PATCH and DELETE.
```

Comment:
We need to go with the original here for two reasons.
- The capitalized versions of GET, POST, etc. are the HTTP verbs, not clauses for our match function.
- If you count, there are only five verbs listed, and we mention eight clauses. :^)